src/views/Employers/EmployerComponents/EmploymentConnectionForm.jsx

```diff
@@ -379,6 +380,19 @@ const EnrollmentConnectionForm = (props) => {
                       placeholder="Salary Offered"
                     />
                   </div>
+                  {showEndDate && (
```

Comment:
there are total three EmploymentConnectionForms in the code:
1. Student Details page
2. Employer Details page
3. Oppportunity Details page

You have currently updated the form that shows up in the Employer Detail page and this is fine. But I believe you will have to make changes to the other two forms as well.

Reply:
Sure, Thanks,

I have made the same changes for the Student Details page and for the Opportunity Details page.